jwt/__init__.py

```diff
@@ -7,11 +7,11 @@
 from __future__ import unicode_literals
 import base64
 import binascii
+import datetime
 import hashlib
 import hmac
 import sys
 
-from datetime import datetime
```

Comment:
Specifically, it doesn't make sense to add `datetime.` in 10+ places when we can just add timedelta to this import.